vagrant/README.md

```diff
@@ -0,0 +1,29 @@
+# Vagrant
+
+Here you will find a `Vagrantfile` to build a development environment for `kvm-vmi`.
+
+# Requirements
+
+- `vagrant`
+- `vagrant-libvirt` plugin
+- `ansible >= 2.2.1.0` (use a virtualenv, install it via pip)
+
+# Setup
+
+At first, make sure to run `vagrant plugin update vagrant-libvirt` to have
```

Comment:
It's usually better to be less verbose and showing the sequence of commands.

Example:

> Update vagrant to the latest version:
> ```$ vagrant plugin update vagrant-libvirt```